.travis/publish_docs.sh

```diff
@@ -88,13 +102,21 @@ cp -r target/staging/* "$release_tag"/
 
 printf "${CYAN}Creating redirecting index.html.$RESET \n"
 mv target/build_index.sh "."
-sh ./build_index.sh "$release_tag"
+bash ./build_index.sh "$release_tag"
```

Comment:
The problem is not with the script itself. Its how it is being executed if I am understanding it correctly. To be explicit with the problem - The shell that travis loads doesn't have permission to execute this script. Why ? I don't know. Its definitely not with the permissions on the file and I don't want to run the build with sudo. 

Here is the [failed build|https://travis-ci.org/cerner/beadledom/builds/193790722] (open the `deploying application` stage in the log to see the error). Here is the immediate [build that passed|https://travis-ci.org/cerner/beadledom/builds/193797533]. Here is the diff between the two builds - https://github.com/cerner/beadledom/compare/7e293aaadf68...7cefc4f57d35